core/src/main/scala/org/bitcoins/core/util/NumberUtil.scala

```diff
@@ -336,6 +336,16 @@ sealed abstract class NumberUtil extends BitcoinSLogger {
   def posInt: Int = {
     Math.abs(scala.util.Random.nextInt())
   }
+
+  /** From 0th place to numDigits place */
```

Comment:
a more descriptive scaladoc please. Something like "Decomposes the given num into a ....." ? Also linking to the spec here in the future would be great.